tests/test_avss_value_processor.py

```diff
@@ -319,3 +323,42 @@
 
         # This is set by another method and should not have been updated.
         assert all(proc.next_idx_to_return_per_dealer[i] == 0 for i in range(n))
+
+
+@mark.parametrize(
+        "n, t, input_next_ids, output_next_ids, per_dealer_input, output_queue_vals", (
```

Comment:
Can you include a guide of how to read this? It looks daunting

Reply:
Done